modules/php/utils.php

```diff
@@ -197,7 +197,7 @@ function setupCards(int $playerCount) {
             $number = $animal->cardsByGender[$playerCount];
 
             if ($useFrog && $type == 1) { $number -= 2; } // frog remove a couple of snails
-            if ($useCrocodile && $type == 3) { $number -= 1; } // crocodile remove a couple of donkeys
+            if ($useCrocodile && $type == 3) { $number -= 2; } // crocodile remove a couple of donkeys
```

Comment:
If you look at how it's created just after, except for snails, there are a couple generated for each number, so no mistake here